.gitignore

```diff
@@ -13,3 +13,6 @@ config/local*
 tags
 # Vim
 *.swp
+# Ingest JSON files
```

Comment:
This wouldn't block merging this PR, but maybe it would be good to just add a utility directory of something like /ingest_json to the .gitignore so that we can add, remove, or rename files as needed without worrying about adding or removing them individually to the gitignore

Reply:
Sounds like a good idea. I'll implement that as a commit.